unpackaged/pre/case_behavior_record_types/objects/Case.object

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <businessProcesses>
+        <fullName>Behavior Management</fullName>
+        <isActive>true</isActive>
+        <values>
+            <fullName>Closed</fullName>
+            <default>false</default>
+        </values>
+        <values>
+            <fullName>Escalated</fullName>
+            <default>false</default>
+        </values>
+        <values>
+            <fullName>New</fullName>
+            <default>true</default>
+        </values>
+        <values>
+            <fullName>Working</fullName>
+            <default>false</default>
+        </values>
+    </businessProcesses>
+    <recordTypes>
+        <fullName>Incident</fullName>
```

Comment:
When this was moved out of the package, it looks like the Location and Category picklist values available to the record type got removed. Should they be added back here? (See https://github.com/SalesforceFoundation/EDA/pull/884/commits/99c930658857aa3187ad99bf5bd4617fecdbc419#diff-e0dc43ce6e02eabdfe8f745916f6385bL89)

Reply:
I am wondering if we should leave those picklist values in src/objects/Case.object?

Reply:
@xinnanlisf @lifewithryan I'm talking about the picklist values available for each record type that was removed in this commit: https://github.com/SalesforceFoundation/EDA/commit/99c930658857aa3187ad99bf5bd4617fecdbc419#diff-e0dc43ce6e02eabdfe8f745916f6385bL89 -- the picklist values still exist in src/objects/Case.object, but we've lost the record type assignment.

Reply:
ooooohhhhh, gotcha good catch @robertbwatson -- yea we should probably specify the the picklist values in the recordtype metadata as well, otherwise they won't be available to said recordtype -- or maybe they're available for all because I did see them in my test.

Reply:
We want those picklist values only unmanaged? We can't have them both in managed and unmanaged otherwise the build will fail..

Reply:
I thought we could have them be managed but the unmanaged bit just assigns the values to the correct record type? Meaning in src/objects/Case.object we have the "field" and the valueset defined, but in unpackaged/.../recordtypes/Case.object we have the recordType metadata and the values listed for that? Are we saying that makes the build fail? Could have sworn we did that elsewhere.

Reply:
Yeah once a record type is created all the picklist fields assigned will be moved under the new record type.

Reply:
@xinnanlisf  @lifewithryan  The build failure: "Update of RecordType Case.Incident: Error on line 29, col 25: Global value set 'Category__c' in record type 'Incident' on the entity 'Case' cannot be resolved" suggests that it's looking for a global value set. I think this is because this is in /pre/ - so it's running *before* the fields actually exist on the Case object.

Reply:
The record type picklist assignment probably needs to happen in /post/